ui/lib/pki/addon/components/pki-issuer-cross-sign.js

```diff
@@ -0,0 +1,145 @@
+ * issuer into an existing intermediate mount using three inputs from the user:
+ * intermediateMount (the mount where the issuer to be cross signed lives)
+ * intermediateIssuer (the name of the intermediate issuer, located in the above mount)
+ * newCrossSignedIssuer (the name of the to-be-cross-signed intermediate issuer)
+ *
+ * The requests involved and how those inputs are used:
+ * 1. Read an existing intermediate issuer
+ *    -> GET /:intermediateMount/issuer/:intermediateIssuer
+ * 2. Create a new CSR based on this existing issuer ID
+ *    -> POST /:intermediateMount/intermediate/generate/existing
+ * 3. Sign it with the new parent issuer, minting a new certificate.
+ *    -> POST /this.args.parentIssuer.backend/issuer/this.args.parentIssuer.issuerName/sign-intermediate
+ * 4. Import it back into the existing mount
+ *    -> POST /:intermediateMount/issuers/import/bundle
+ * 5. Read the imported issuer
+ *    -> GET /:intermediateMount/issuer/:issuer_id
+ * 5. Update this issuer with the newCrossSignedIssuer
+ *    -> POST /:intermediateMount/issuer/:issuer_id
+ *
+ * @example
+ * ```js
+ * <PkiIssuerCrossSign @parentIssuer={{this.model}} />
+ * ```
+ * @param {object} parentIssuer - the model of the issuing certificate that will sign the issuer to-be cross-signed
+ */
+
+export default class PkiIssuerCrossSign extends Component {
+  @service store;
+  @tracked formData = [];
+  @tracked signedIssuers = [];
+
+  inputFields = [
+    {
+      label: 'Mount path',
+      key: 'intermediateMount',
+      placeholder: 'Mount path',
+      helpText: 'The mount in which your new certificate can be found.',
+    },
+    {
+      label: "Issuer's current name",
+      key: 'intermediateIssuer',
+      placeholder: 'Current issuer name',
+      helpText: 'The API name of the previous intermediate which was cross-signed.',
+    },
+    {
+      label: 'New issuer name',
+      key: 'newCrossSignedIssuer',
+      placeholder: 'Enter a new issuer name',
+      helpText: `This is your new issuer’s name in the API.`,
+    },
+  ];
+
+  @task
+  @waitFor
+  *submit(e) {
+    e.preventDefault();
+    this.signedIssuers = [];
+
+    // iterate through submitted data and cross-sign each certificate
+    for (let row = 0; row < this.formData.length; row++) {
+      const { intermediateMount, intermediateIssuer, newCrossSignedIssuer } = this.formData[row];
+      try {
+        // returns data from existing and newly cross-signed issuers
+        // { intermediateIssuer: existingIssuer, newCrossSignedIssuer: crossSignedIssuer, intermediateMount: intMount }
+        const data = yield this.crossSignIntermediate(
+          intermediateMount,
+          intermediateIssuer,
+          newCrossSignedIssuer
+        );
+        this.signedIssuers.addObject({ ...data, hasError: false });
+      } catch (error) {
+        this.signedIssuers.addObject({ ...this.formData[row], hasError: errorMessage(error) });
+        continue;
```

Comment:
I think it will automatically move to the next iteration since you are catching the error so shouldn't need to continue.